Exercise_B/index.css

```diff
@@ -2,3 +2,26 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+.blue{
+    background-color: blue;
+    float:left;
+    width: 100px;
+    height:100px;
+    margin: 10px;
+}
+.red{
+    background-color: red;
+    clear: both;
+    width: 100px;
+    height:100px;
+    margin: 10px;
+    margin-left: 45%;
+}
+.green{
+    background-color: green;
+    float:right;
+    width: 100px;
+    height:100px;
+    margin: 10px;
```

Comment:
I would extract the width and height into a separate class such that you do not repeat the properties over and over again for each block.

Use an ID for the colour and float.
Make a new class that is applied to every html element for the height and width.

But well done and superb use of float and clear. To improve this code you could alter you red block and use margin auto to center the element